speedscale/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Speedscale",
+  "maintainer": "support@speedscale.com",
+  "manifest_version": "1.0.0",
+  "name": "speedscale",
+  "metric_prefix": "speedscale.",
+  "metric_to_check": "",
+  "creates_events": true,
+  "short_description": "",
```

Comment:
This is a required field - it's a brief description of your offering that will be displayed on the tile.